src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -102,6 +105,7 @@ public Config validate() {
       validateMaxBufferedRecords();
       validateProxy();
       validateSsl();
+      validateVersion(client);
```

Comment:
i would validate the version after validating the connection because if we can't establish a connection theres no point in checking the version